tests/rptest/tests/cloud_storage_timing_stress_test.py

```diff
@@ -247,6 +247,11 @@ def __init__(self, test_context):
         self.admin = Admin(self.redpanda)
         self.checks = []
 
+        # Fine tune the allowed overshoot for debug builds as they are
```

Comment:
RedpandaTest has a `debug_mode` attribute for checking the build type directly.

Maybe we should just do a `@skip_debug_mode` on this test though?  That's what I mostly do with tests that require any significant throughput, as it's so hard to predict just how slow a debug build will be when run in the noisy docker environment.

Reply:
> RedpandaTest has a debug_mode attribute for checking the build type directly

How did I miss this? Will update!

> Maybe we should just do a @skip_debug_mode on this test though? That's what I mostly do with tests that require any significant throughput, as it's so hard to predict just how slow a debug build will be when run in the noisy docker environment.

This test doesn't really have significant throughput requirements though. Also, in the stabilisation phase of this test I noticed that debug and release builds surfaced different failure modes. I'd be inclined to keep the debug build for now.
If it proves to still be flaky in the future we can get rid.